syntax/encoding/testdata/yaml/basic/doc.yaml

```diff
@@ -0,0 +1,20 @@
+# Here's a comment
+"null-value": null
+boolean-value: true
+integer-value: 42
+number-value: 3.14
+string-value: hello, world
+array-value:
+  - some
```

Comment:
Would it be worth testing the other array syntax as well? e.g., `array-value: ["some", "array", "entries"]`

Reply:
Yes, I'll add that (and a case for flow-style object literals)